src/vim9class.c

```diff
@@ -183,9 +183,12 @@ add_members_to_class(
  * "cl" implementing that interface.
  */
     int
-object_index_from_itf_index(class_T *itf, int is_method, int idx, class_T *cl)
+object_index_from_itf_index(class_T *itf, int is_method, int idx, class_T *cl,
+								int is_static)
 {
+    // TODO: should the following be ">="
     if (idx > (is_method ? itf->class_obj_method_count
+				   : is_static ? itf->class_class_member_count
 						: itf->class_obj_member_count))
     {
```

Comment:
Looks like a bug. If reviewer(s) agree, I'll make the change.

Reply:
Made the change, didn't want to lose it. This PR should be ready to review/go without changes